class-wpcom-compat-command.php

```diff
@@ -76,6 +76,105 @@ function import_protected_embeds( $args ) {
 			WP_CLI::line( sprintf( 'Failed to insert %d embeds', $errors ) );
 		}
 	}
+
+	/**
+	 * Import theme options, sidebars options and widgets options from a JSON file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <file>
+	 * : The JSON file with all the settings.
+	 *
+	 * [--dry-run=<true>]
+	 * : Do a "dry run" and no data modification will be done.  Defaults to true.
+	 *
+	 * [--yes]
+	 * : Skip the confirmation prompt
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Generates the JSON file with theme options (on WordPress.com sandbox)
+	 *     $ wp vip-export theme-options --url=example.com --filename=file.json
+	 *
+	 *     # Copies theme options from the JSON file.
+	 *     $ wp wpcom-compat import-theme-options file.json
+	 *
+	 *
+	 * @subcommand import-theme-options
+	 */
+	function import_theme_options($args, $assoc_args) {
+		$dry_run = WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run', true );
+		$filename = $args[0];
+		// Force a boolean, always default to true.
+		$dry_run = filter_var( $dry_run, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE ) ?? true;
+		if ( $dry_run ) {
+			WP_CLI::warning( 'Performing a dry run, with no database modification.' );
+		}
+		$current_theme = get_option( 'stylesheet' );
```

Comment:
Should we have a check to make sure the theme still isn't prefixed (`vip/*`) before running?  Would it make a difference?